documentation/modules/con-jmxtrans.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+[id='con-jmxtrans-{context}']
+= Jmxtrans
+
+Jmxtrans reads JMX data from a secure or insecure Kafka brokers and pushes data to remote sinks in various data formats.
```

Comment:
_Suggested change:_

Jmxtrans reads JMX data from a secure or insecure Kafka brokers and pushes data to remote sinks in various data formats.

`>>`

JmxTrans reads JMX metrics data from secure or insecure Kafka brokers and pushes the data to remote sinks in various data formats.